great_expectations/checkpoint/actions.py

```diff
@@ -1032,32 +1033,30 @@ def _run(
         )
 
 
+@public_api
+@deprecated_argument(
+    argument_name="target_site_names",
+    version="0.10.10",
+    message="target_site_names is deprecated as of v0.10.10 and will be removed in v0.16. Please use site_names instead.",
+)
 class UpdateDataDocsAction(ValidationAction):
-    """
-    UpdateDataDocsAction is a validation action that
-    notifies the site builders of all the data docs sites of the Data Context
-    that a validation result should be added to the data docs.
-
-    **Configuration**
-
-    .. code-block:: yaml
```

Comment:
Code Blocks work now!! You can try the following. There is now an example in the [readme](https://github.com/great-expectations/great_expectations/blob/develop/docs/sphinx_api_docs_source/README.md) as well.
````
```yaml
- name: update_data_docs
action:
  class_name: UpdateDataDocsAction
```
````